webapp/src/types/types.ts

```diff
@@ -22,6 +22,17 @@ export type ChannelState = {
     enabled?: boolean;
 }
 
+export type CallsClientJoinData = {
+    channelID: string;
+    title?: string;
+    threadID?: string;
+
+    // Calls bot only
+    // contextID is the id used to track the context of the bot connection to
+    // a call (e.g. recording, transcription).
+    contextID?: string;
+}
```

Comment:
I think this could fit in https://github.com/mattermost/calls-common since it's part of the public API. For this PR though it's practically unnecessary.